app/src/main/kotlin/org/equeim/tremotesf/ui/addtorrent/AddTorrentFileFragment.kt

```diff
@@ -371,14 +385,25 @@ class AddTorrentFileFragment : AddTorrentFragment(
                     savedInstanceState
                 )
 
-                startDownloadingCheckBox.isChecked = GlobalRpc.serverSettings.startAddedTorrents
+                savedInstanceState?.getBoolean(START_DOWNLOADING_CHECKBOX_STATE_KEY)?.let {
```

Comment:
AFAIK CheckBox already saves its state internally. Maybe we should just set initial state if savedInstanceState == null and don't touch it otherwise?

Reply:
Seems so, I'll force push in a bit~

Reply:
See: 903d29d4a61bbfd1033532508f2c1df9d6d114f5, I only did it for AddTorrentFileFragment since AddTorrentLinkFragment was already doing similar thing with setInitialTorrentLink check.